src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -153,6 +154,60 @@ public void VerifyMultiTFMInConsoleTemplate(DotNetLanguage language)
         newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot, GetFrameworks);
     }
 
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    [InlineData(DotNetLanguage.FSharp)]
+    public void VerifyMVCTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Mvc,
+            DotNetSdkActions.Build | DotNetSdkActions.Run | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    [InlineData(DotNetLanguage.FSharp)]
+    public void VerifyWebAPITemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.WebApi,
+            DotNetSdkActions.Build | DotNetSdkActions.Run | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    [InlineData(DotNetLanguage.FSharp)]
+    public void VerifyWebTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Web,
+            DotNetSdkActions.Build | DotNetSdkActions.Run | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    public void VerifyBlazorWasmTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.BlazorWasm,
+            DotNetSdkActions.Build | DotNetSdkActions.Run | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+
+    [Fact]
+    [Trait("Category", "Offline")]
+    public void VerifyPreMadeSolution()
+    {
+        string path = "test" + Path.DirectorySeparatorChar + "resources" + Path.DirectorySeparatorChar + "SampleProject";
```

Comment:
Since the pre-made solution isn't really an SDK template, I wonder if it's time to add a new test project (e.g. `Microsoft.DotNet.ScenarioTests.Something`)

Reply:
Yeah, you're not wrong. I was avoiding it as it was the only exception so far but I can work on making the change.

Reply:
Since it's only a single test so far, I'd say get it working in the current infra and then move it over.